listenbrainz/listenstore/tests/test_timescalelistenstore.py

```diff
@@ -0,0 +1,59 @@
+# coding=utf-8
+
+import listenbrainz.db.user as db_user
+import os
+import time
+import ujson
+from time import sleep
+import logging
+
+from listenbrainz.db.testing import DatabaseTestCase
+from listenbrainz.db import timescale as ts
+from listenbrainz import config
+from listenbrainz.listenstore.tests.util import create_test_data_for_influxlistenstore, generate_data
+from listenbrainz.webserver.timescale_connection import init_ts_connection
+
+
+TIMESCALE_SQL_DIR = "/code/listenbrainz/admin/timescale"
```

Comment:
This needs to be a relative path. See this file for examples: https://github.com/metabrainz/listenbrainz-server/blob/52705348/listenbrainz/db/__init__.py#L1

Reply:
Yeah. I will fix that. I was just trying to get it started.